api/src/test/java/org/openmrs/api/db/PatientDAOTest.java

```diff
@@ -2033,4 +2033,33 @@ public void getPatients_shouldGetNoVoidedPersonWhenVoidedFalseIsPassed() throws
 		List<Patient> patients = dao.getPatients("voided-bravo", false, 0, 11);
 		Assert.assertEquals(0, patients.size());
 	}
+	/**
```

Comment:
@teleivo : Yes there are lot more test cases with permutation and combination of various attributes selected for duplicate matching as well. However since there were no cases initially when I picked this up, wanted to add the cases to cover the scenarios for which this fix was being done, we shall create more cases for all positive and negative scenarios.